ADB_world/Worldmap_interactive/Main_interactive.py

```diff
@@ -0,0 +1,181 @@
+#######################################
+# IMPORTS #############################
+#######################################
+
+import plotly.graph_objects as go
+import sys
+import os
+
+# Get the directory of the current file
+current_directory = os.path.dirname(__file__)
```

Comment:
Please use the more current `Path` library: https://docs.python.org/3/library/pathlib.html#pathlib.Path.cwd